src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 50px;
+}
+
+.nav__list {
+  list-style: none;
+  width: auto;
+  margin: 0 50px 0 0;
```

Comment:
your test didn't pass because there shouldn't be any margins on your `nav` element, your header should have 50px horizontal padding instead